subsys/bluetooth/host/audio/endpoint.c

```diff
@@ -362,31 +366,33 @@ static void ep_config(struct bt_audio_ep *ep, struct net_buf_simple *buf)
 	pref->pref_pd_max = sys_get_le24(cfg->prefer_pd_max);
 
 	BT_DBG("dir 0x%02x framing 0x%02x phy 0x%02x rtn %u latency %u "
-	       "pd_min %u pd_max %u codec 0x%02x ", ep->stream->cap->type,
+	       "pd_min %u pd_max %u codec 0x%02x ", stream->cap->type,
 	       pref->framing, pref->phy, pref->rtn, pref->latency, pref->pd_min,
-	       pref->pd_max, ep->stream->codec->id);
+	       pref->pd_max, stream->codec->id);
 
 	bt_audio_ep_set_codec(ep, cfg->codec.id,
 			      sys_le16_to_cpu(cfg->codec.cid),
 			      sys_le16_to_cpu(cfg->codec.vid),
 			      buf, cfg->cc_len, NULL);
 
 	/* Notify local capability */
-	if (ep->cap && ep->cap->ops && ep->cap->ops->reconfig) {
-		ep->cap->ops->reconfig(ep->stream, ep->cap, ep->stream->codec);
+	if (stream->ops != NULL && stream->ops->configured != NULL) {
```

Comment:
Line above: Is it still correct to say the we notify capabilites? (Other places as well.)

Reply:
Updated in a new commit.